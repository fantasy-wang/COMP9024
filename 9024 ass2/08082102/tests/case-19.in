ail aim aril armil hail haul haulm mail main maint nim nimb nimbi pail pair pais paisa parr part pass peart pirr saim saip salp salt sam same samp san sank sar sare sarip sart sat save sax say saya scrip sere sero sert serut shrip shy six skat skate skater sky slank slink slinky slip snip snipy sore sot sow sowl sown sowt sowte spank spate sprank sprink stave stow stre stret strip strix suave swamp swat sway sweat taint taunt taut tax taxi taxy thrip tot toto trip tryp tux twat vain wamp wasp wast were whamp yowt
