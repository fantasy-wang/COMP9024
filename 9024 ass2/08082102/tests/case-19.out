Dictionary
0: ail
1: aim
2: aril
3: armil
4: hail
5: haul
6: haulm
7: mail
8: main
9: maint
10: nim
11: nimb
12: nimbi
13: pail
14: pair
15: pais
16: paisa
17: parr
18: part
19: pass
20: peart
21: pirr
22: saim
23: saip
24: salp
25: salt
26: sam
27: same
28: samp
29: san
30: sank
31: sar
32: sare
33: sarip
34: sart
35: sat
36: save
37: sax
38: say
39: saya
40: scrip
41: sere
42: sero
43: sert
44: serut
45: shrip
46: shy
47: six
48: skat
49: skate
50: skater
51: sky
52: slank
53: slink
54: slinky
55: slip
56: snip
57: snipy
58: sore
59: sot
60: sow
61: sowl
62: sown
63: sowt
64: sowte
65: spank
66: spate
67: sprank
68: sprink
69: stave
70: stow
71: stre
72: stret
73: strip
74: strix
75: suave
76: swamp
77: swat
78: sway
79: sweat
80: taint
81: taunt
82: taut
83: tax
84: taxi
85: taxy
86: thrip
87: tot
88: toto
89: trip
90: tryp
91: tux
92: twat
93: vain
94: wamp
95: wasp
96: wast
97: were
98: whamp
99: yowt
Ordered Word Ladder Graph
V=100, E=144
<0 1> <0 2> <0 4> <0 7> <0 13> 
<1 0> <1 10> <1 22> 
<2 0> <2 3> 
<3 2> 
<4 0> <4 5> <4 7> <4 13> 
<5 4> <5 6> 
<6 5> 
<7 0> <7 4> <7 8> <7 13> 
<8 7> <8 9> <8 93> 
<9 8> <9 80> 
<10 1> <10 11> 
<11 10> <11 12> 
<12 11> 
<13 0> <13 4> <13 7> <13 14> <13 15> 
<14 13> <14 15> <14 17> 
<15 13> <15 14> <15 16> <15 19> 
<16 15> 
<17 14> <17 18> <17 21> 
<18 17> <18 20> <18 34> 
<19 15> 
<20 18> 
<21 17> 
<22 1> <22 23> <22 26> 
<23 22> <23 24> <23 28> <23 33> <23 55> <23 56> 
<24 23> <24 25> <24 28> 
<25 24> <25 34> <25 35> 
<26 22> <26 27> <26 28> <26 29> <26 31> <26 35> <26 37> <26 38> 
<27 26> <27 28> <27 32> <27 36> 
<28 23> <28 24> <28 26> <28 27> <28 76> <28 94> 
<29 26> <29 30> <29 31> <29 35> <29 37> <29 38> 
<30 29> <30 52> <30 65> 
<31 26> <31 29> <31 32> <31 34> <31 35> <31 37> <31 38> 
<32 27> <32 31> <32 34> <32 36> <32 41> <32 58> <32 71> 
<33 23> <33 40> <33 45> <33 73> 
<34 18> <34 25> <34 31> <34 32> <34 35> <34 43> 
<35 25> <35 26> <35 29> <35 31> <35 34> <35 37> <35 38> <35 48> <35 59> <35 77> 
<36 27> <36 32> <36 69> <36 75> 
<37 26> <37 29> <37 31> <37 35> <37 38> <37 47> <37 83> 
<38 26> <38 29> <38 31> <38 35> <38 37> <38 39> <38 46> <38 51> <38 78> 
<39 38> 
<40 33> <40 45> <40 73> 
<41 32> <41 42> <41 43> <41 58> <41 71> <41 97> 
<42 41> <42 43> 
<43 34> <43 41> <43 42> <43 44> 
<44 43> 
<45 33> <45 40> <45 73> <45 86> 
<46 38> <46 51> 
<47 37> 
<48 35> <48 49> <48 77> 
<49 48> <49 50> <49 66> 
<50 49> 
<51 38> <51 46> 
<52 30> <52 53> <52 65> 
<53 52> <53 54> 
<54 53> 
<55 23> <55 56> 
<56 23> <56 55> <56 57> 
<57 56> 
<58 32> <58 41> <58 71> 
<59 35> <59 60> <59 63> <59 87> 
<60 59> <60 61> <60 62> <60 63> <60 70> 
<61 60> <61 62> <61 63> 
<62 60> <62 61> <62 63> 
<63 59> <63 60> <63 61> <63 62> <63 64> <63 99> 
<64 63> 
<65 30> <65 52> <65 67> 
<66 49> 
<67 65> <67 68> 
<68 67> 
<69 36> <69 75> 
<70 60> 
<71 32> <71 41> <71 58> <71 72> 
<72 71> 
<73 33> <73 40> <73 45> <73 74> <73 89> 
<74 73> 
<75 36> <75 69> 
<76 28> <76 94> 
<77 35> <77 48> <77 78> <77 79> <77 92> 
<78 38> <78 77> 
<79 77> 
<80 9> <80 81> 
<81 80> <81 82> 
<82 81> 
<83 37> <83 84> <83 85> <83 91> 
<84 83> <84 85> 
<85 83> <85 84> 
<86 45> <86 89> 
<87 59> <87 88> 
<88 87> 
<89 73> <89 86> <89 90> 
<90 89> 
<91 83> 
<92 77> 
<93 8> 
<94 28> <94 76> <94 95> <94 98> 
<95 94> <95 96> 
<96 95> 
<97 41> 
<98 94> 
<99 63> 
Longest ladder length: 15
Longest ladders:
 1: ail -> aim -> saim -> sam -> san -> sar -> sare -> sart -> sat -> sot -> sow -> sowl -> sown -> sowt -> sowte
 2: ail -> aim -> saim -> sam -> san -> sar -> sare -> sart -> sat -> sot -> sow -> sowl -> sown -> sowt -> yowt
 3: ail -> hail -> mail -> pail -> pair -> parr -> part -> sart -> sat -> sot -> sow -> sowl -> sown -> sowt -> sowte
 4: ail -> hail -> mail -> pail -> pair -> parr -> part -> sart -> sat -> sot -> sow -> sowl -> sown -> sowt -> yowt
